contracts/evoting/mod.go

```diff
@@ -19,6 +21,33 @@ import (
 	_ "github.com/dedis/d-voting/contracts/evoting/json"
 )
 
+var (
+	PromElectionStatus = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "dvoting_evoting_status",
+		Help: "status of election",
+	})
```

Comment:
likewise, here and elsewhere, it would have to look something like
```suggestion
	PromElectionStatus = prometheus.NewGauge(prometheus.GaugeOpts{
		Name: "dvoting_evoting_status",
		Help: "status of election",
	},
	[]string{"election"})
```